### PR TITLE
Update german (de) localization

### LINKED ISSUE #646: Improved German localization
Hi Erik!
Is it possible, to translate "Actively Exploited CVEs" to german on the left side of Nudge to "Aktiv ausgenutzte CVEs"?

thx

### PR DESCRIPTION
**Pull request by northalpha**
Using informal speach like Apple does talking about their customer and their products

This will also will fix #646

### CODE REVIEW

File: Localizable.xcstrings [hunk 1]
```diff
@@ -546,7 +546,7 @@
         "de" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Um sicherzustellen, dass die IT Ihr Gerät ordnungsgemäß schützen kann, ist ein vollständig aktuelles Gerät erforderlich.\n\nWenn Sie kein Ersatzgerät erhalten, verlieren Sie den Zugriff auf einige Dinge, die Sie für Ihre täglichen Aufgaben benötigen.\n\nFür weitere Informationen hierzu klicken Sie bitte auf die Schaltfläche **Gerät ersetzen**."
+            "value" : "Um sicherzustellen, dass die IT dein Mac ordnungsgemäß schützen kann, ist ein aktueller Mac erforderlich.\n\nWenn Du kein Ersatzgerät erhalten hast, verlierst Du ggf. den Zugriff, die Du für deine täglichen Aufgaben benötigst.\n\nFür weitere Informationen hierzu klicke bitte auf die Schaltfläche **Mac ersetzen**."
```
Comment (AirBookMac) [inline Localizable.xcstrings:549]: deinen Mac

File: Localizable.xcstrings [hunk 2]
```diff
@@ -654,7 +654,7 @@
         "de" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Ein vollständig aktualisiertes Gerät ist erforderlich, um sicherzustellen, dass die IT-Abteilung dein Gerät effektiv schützen kann.\n\nWenn du dein Gerät nicht aktualisierst, verlierst du möglicherweise den Zugriff auf einige Werkzeuge, die du für deine täglichen Aufgaben benötigst.\n\nUm das Update zu starten, klicke auf die Schaltfläche **Gerät Aktualisieren** und befolge die angegebenen Schritte."
+            "value" : "Ein vollständig aktualisierter Mac ist erforderlich, um sicherzustellen, dass die IT-Abteilung dein Mac effektiv schützen kann.\n\nWenn du dein Mac nicht aktualisierst, verlierst du möglicherweise den Zugriff auf einige Werkzeuge, die du für deine täglichen Aufgaben benötigst.\n\nUm das Update zu starten, klicke auf die Schaltfläche **Mac aktualisieren** und befolge die angegebenen Schritte."
```
Comment (AirBookMac) [inline Localizable.xcstrings:657]: die IT-Abteilung dein Mac effektiv -> die IT-Abteilung deinen Mac effektiv

File: Localizable.xcstrings [hunk 3]
```diff
@@ -2922,7 +2922,7 @@
         "de" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Bitte aktualisiere dein Gerät, um dieses Programm zu verwenden."
+            "value" : "Bitte aktualisiere dein Mac, um dieses Programm zu verwenden."
```
Comment (AirBookMac) [inline Localizable.xcstrings:2925]: Bitte aktualisiere dein Mac -> Bitte aktualisiere deinen Mac

**Review by AirBookMac — CHANGES_REQUESTED** [review]
dein Mac -> deinen Mac

File: Localizable.xcstrings (hunk 1 above)
Comment (northalpha) [reply Localizable.xcstrings]: fixed

**Review by AirBookMac — COMMENTED** [review]
german language looks fine

**Pull request merged**